### ISSUE TITLE
Usage of Tesla streaming API?

### ISSUE THREAD

**Issue by bastelbert** [onokje/node-red-contrib-tesla#50]
Hi, first of all thank you very much for this great project! I already used your node to realize a charge controller based on my solar production. 

I am wondering if it's possible to integrate the Tesla streaming API to receive streaming vehicle data on-change, like these two projects do?
    1. https://github.com/adriankumpf/teslamate
        https://docs.teslamate.org/docs/faq#why-am-i-missing-data-when-not-using-the-streaming-api
    2. https://pypi.org/project/TeslaPy/

**Comment by onokje** [comment]
Yeah I am definitely interested in the streaming api. I have looked into it before, but that was a few years ago. There weren't any examples yet :)
I will look into it in the future. In the mean time, ideas and examples are welcome.

**Comment by bastelbert** [comment]
Maybe it's worth a shot taking a look a Teslamate?
